### PR TITLE
Update build to add PHPCS and PHPStan

### PR DESCRIPTION
**Pull request by lcobucci**

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="Coding Standards for doctrine">
+    <description>Coding Standards for doctrine.</description>
```
Comment (Majkl578) [inline phpcs.xml.dist]: This should probably be _"Coding Standard for Doctrine Collections."_ (and same for ruleset name)

File: phpunit.xml.dist [hunk 2]
```diff
@@ -1,13 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-         backupGlobals="false"
-         colors="true"
-         verbose="true"
-         beStrictAboutOutputDuringTests="true"
-         beStrictAboutTestsThatDoNotTestAnything="true"
```
Comment (Majkl578) [inline phpunit.xml.dist]: Why was this removed?

Comment (lcobucci) [reply phpunit.xml.dist]: I've tried explained on the commit message (cb7f39d) but the reason is that we were just repeating the default configuration, which is kind of redundant.

File: phpcs.xml.dist (hunk 1 above)
Comment (lcobucci) [reply phpcs.xml.dist]: You're right, I was too lazy to change it (and I probably didn't change on the other projects as well - shame on me!).

Comment (lcobucci) [reply phpcs.xml.dist]: I've just found out that the `name` attribute and the `description` element are useless here. So I'll just kill them
